packages/repo/src/repo.ts

```diff
@@ -31,6 +31,7 @@ export class Repo extends ReadableRepo {
 
   constructor(params: Params) {
     super(params)
+    this.storage = params.storage
```

Comment:
Seemed slightly unrelated, just wanted to make sure it landed in here intentionally.